pkg/manager/manager.go

```diff
@@ -173,6 +184,7 @@ func (r RunnableFunc) Start(s <-chan struct{}) error {
 }
 
 // New returns a new Manager for creating Controllers.
+// nolint: gocyclo
```

Comment:
??

Reply:
please don't just turn off the lints because they're annoying.  This seems to suggest that perhaps leader election setup should be in a separate helper function.